records_mover/db/mysql/mysql_db_driver.py

```diff
@@ -15,13 +15,30 @@
                              FLOAT64_SIGNIFICAND_BITS,
                              num_digits)
 from ..driver import DBDriver
-from typing import Optional, Tuple
+from ..loader import LoaderFromFileobj, LoaderFromRecordsDirectory, NegotiatesLoadFormatImpl
+from typing import Optional, Tuple, Union
 
 
 logger = logging.getLogger(__name__)
 
 
-class MySQLDBDriver(DBDriver):
+class MySQLDBDriver(DBDriver,
+                    LoaderFromFileobj,
+                    LoaderFromRecordsDirectory,
+                    NegotiatesLoadFormatImpl):
+    def loader(self) -> Optional[Union[LoaderFromFileobj, LoaderFromRecordsDirectory]]:
+        return None
+
+    # TODO: There's a defacto constraint that you could have either
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker